### PR TITLE
information on hooks for hosted agent'

### PR DESCRIPTION
**Pull request by mcncl**

### CODE REVIEW

**Comment by buildkite-systems** [comment]
Preview URL: https://5c445b648016243db82a38caa957d4a0eaba9aa5--bk-docs-preview.netlify.app

File: pages/agent/v3/hooks.md [hunk 1]
```diff
@@ -206,3 +206,6 @@ An example of a Windows `environment.bat` hook:
 ECHO "--- \:house_with_garden\: Setting up the environment"
 SET GITHUB_RELEASE_ACCESS_KEY='xxx'
 ```
+### Hooks on hosted agents
+
+Currently, hooks located on the agent are not available on hosted agents. Instead we recommend using plugin or repository based hooks.
```
Comment (IcyWednesdays) [inline pages/agent/v3/hooks.md:211]:
```suggestion
Currently, hooks located on the agent are not available on hosted agents. Instead, we recommend using plugin or repository based hooks.
```

**Comment by gilesgas** [comment]
Thanks for your contribution @mcncl ! 🎉

I just want to confirm if your addition applies to all types of Buildkite hosted agents and not Windows hosted agents, since the heading level nesting you've applied makes it appear as though your addition is part of the **Job hooks on Windows** section (preview [here](https://5c445b648016243db82a38caa957d4a0eaba9aa5--bk-docs-preview.netlify.app/docs/agent/v3/hooks#job-hooks-on-windows)).

If this is the case, you'll need to remove to remove a hash/pound symbol from the start of your new heading to promote it. I'll do this in a subsequent suggested commit, along with some minor rewording.

File: pages/agent/v3/hooks.md [hunk 2]
```diff
@@ -206,3 +206,6 @@ An example of a Windows `environment.bat` hook:
 ECHO "--- \:house_with_garden\: Setting up the environment"
 SET GITHUB_RELEASE_ACCESS_KEY='xxx'
 ```
+### Hooks on hosted agents
+
+Currently, hooks located on the agent are not available on hosted agents. Instead, we recommend using plugin or repository based hooks.
```
Comment (gilesgas) [inline pages/agent/v3/hooks.md:211]: Promote heading (as suggested in my comment above) and link through to relevant sections of the docs towards the top of this page (for better context).

```suggestion
## Hooks on hosted agents

Currently, hosted agents do not support [agent hooks](#hook-locations-agent-hooks). Instead, use either [repository](#hook-locations-repository-hooks)- or [plugin](#hook-locations-plugin-hooks)-based hooks.
```

**Pull request merged**